src/queries/queries.js

```diff
@@ -10,6 +10,50 @@ const USERS_QUERY = gql`
   }
 `;
 
+const USER_QUERY = gql`
+  query User($id: ID!) {
+    user(id: $id) {
+      id
+      name
+      email
+    }
+  }
+`;
+
+const CREATE_USER_MUTATION = gql`
+  mutation CreateUser($name: String!, $email: String!, $password: String!, $passwordConfirmation: String!) {
```

Comment:
I really like how yall abstracted the queries into this file, but might I suggest there be a `mutations` file with the gql mutations to keep them separate :)